src/js/containers/keyword/table/ResultsTableContainer.jsx

```diff
@@ -288,15 +292,15 @@ export default class ResultsTableContainer extends React.Component {
     loadColumns() {
         const columns = {};
         tableTypes.forEach((type) => {
-            const allColumns = availableColumns(type.internal);
+            const allColumns = map(availableColumns(type.internal), (data) => data.title);
 
             const parsedColumns = {};
-            allColumns.forEach((title) => {
-                parsedColumns[title] =
+            availableColumns(type.internal).forEach((data) => {
+                parsedColumns[data.title] =
```

Comment:
could do the same thing here as in line 134 of the other `ResultsTableCtonainer` (use `.reduce`)